pkgs/development/tools/misc/bashdb/default.nix

```diff
@@ -1,10 +1,15 @@
 { stdenv, fetchurl }:
 
+let
+  basename = "bashdb-4.4-0.92";
```

Comment:
For very complicated stuff it may be clearer to use explicit recursion (to not lose track of what is being referred to exactly), but it's almost always better to use `rec` if possible.